tests/components/asuswrt/test_sensor.py

```diff
@@ -44,6 +44,27 @@
 MOCK_CURRENT_TRANSFER_RATES = [20000000, 10000000]
 
 
+class MockDevices:
+    """Mock AsusWrt devices."""
+
+    _mock_devices = dict(MOCK_DEVICES)
```

Comment:
If we'd use this class in more than one test the devices would not be reset between the tests.

Instead I'd create a pytest fixture that returns a dict with devices. We can set this dict as return value in the mock and we can modify the contents of the dict as needed in the test. Since it's a pytest fixture that returns the dict, a fresh dict with original content will be created for every test.

Reply:
Not so expert in fixture and mock. Can you provide a reference example?

Reply:
```py
@pytest.fixture(name="mock_devices")
def mock_devices_fixture():
    return {...}
```

Reply:
Sorry but I do not understand how I should modify the content of the dict in the test...

Reply:
Something like this:
```py
def test_something(mock_devices):
    mock_devices.pop("some_existing_device")
    mock_devices["some_other_device"] = "the_other_device"
    ...
```